.drone.star

```diff
@@ -158,7 +158,20 @@ config = {
             "suites": [
                 "cliCommands",
             ],
+        },
+        "apiAppProvider": {
+            "suites": [
+                "apiAppProvider",
+            ],
             "skip": False,
+            "collaborationServiceNeeded": True,
+            "extraCollaborationEnvironment": {
+                "COLLABORATION_APP_NAME": "FakeOffice",
+                "COLLABORATION_APP_ADDR": "http://fakeoffice:8080",
+            },
+            "extraServerEnvironment": {
+                "GATEWAY_GRPC_ADDR": "0.0.0.0:9142",
+            },
```

Comment:
Is it possible to list the required services and use it in the pipeline services
```suggestion
            "extraServerEnvironment": {
                "GATEWAY_GRPC_ADDR": "0.0.0.0:9142",
            },
            "extraServices": [
                fakeOffice,
            ],
```

Reply:
will more changes in this PR. can be done in coming PRs